Tasks/GitHubReleaseV1/package-lock.json

```diff
@@ -1,8 +1,755 @@
 {
   "name": "github.release",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Undo these changes

Reply:
Then I'll need to make it lockFileversion 1?

Reply:
This version2 is backward compatible to v1
https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion